source/index.html.md

```diff
@@ -502,6 +502,16 @@ The following objects can be used as parameters in the *include* query
           "containerMode": ContainerMode,
           "shipmentType": ShipmentType,
           "consolType": ConsolType,
+          "firstArrivalPort": string,
+          "firstArrivalPortUnlocode": string,
+          "firstArrivalPortEta": string,
+          "ownersReference": string,
+          "originEtd": string,
+          "destinationEta": string,
+          "coLoader": string,
+          "coLoaderMblNumber": string,
+          "loadPortEtd": string,
+          "dispatchPortEta": string,
```

Comment:
Can you also add these in the "Files/billOfLading* attributes" area and also in the example of the bol below the attributes

Reply:
The examples of the bol case below the attributes section still missing